x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/UnfollowFollowIndexStep.java

```diff
@@ -23,6 +23,11 @@
         super(key, nextStepKey, client);
     }
 
+    @Override
+    public boolean isRetryable() {
+        return true;
+    }
+
     @Override
     void innerPerformAction(String followerIndex, Listener listener) {
         UnfollowAction.Request request = new UnfollowAction.Request(followerIndex);
```

Comment:
Same comment here about skipping if the index is already unfollowed, a double unfollow throws an error so we'd just be retrying forever if it were executed after the index was already unfollowed